lua/astrocommunity/pack/rust/README.md

```diff
@@ -39,3 +39,62 @@ The first three levels of this are AstronVim-specific configurations, the rest i
     },
   },
 ```
+
+## Use clippy instead of cargo check
+
+`clippy` and `cargo check` are both tools used in the Rust programming ecosystem, but they serve different purposes. Understanding the distinction between the two can help you decide when to use each tool.
+
+### `cargo check`
+
+`cargo check` is a Cargo subcommand that quickly checks your Rust code to ensure it compiles without actually producing an executable. It's much faster than running `cargo build` because it skips the final step of producing a binary. This makes it a useful tool for quickly iterating on your code when you're making lots of changes and want to verify that they compile correctly without waiting for the entire build process to complete.
+
+### `clippy`
+
+`clippy`, on the other hand, is a linter for Rust. It's more comprehensive than `cargo check` because it analyzes your code for common mistakes and idiomatic issues that aren't necessarily compilation errors. `clippy` provides warnings for a wide range of issues, including:
+
+- Code that is unnecessarily complex or verbose
+- Code that can be more idiomatic (more "Rusty")
+- Possible logic errors
+- Performance issues
+- Non-optimal use of Rust's features
+
+### Why Use `clippy` Instead of `cargo check`?
+
+You would use `clippy` instead of (or in addition to) `cargo check` when you want not just to check that your code compiles, but also to ensure that it adheres to best practices and is as clean and efficient as possible. `clippy` can catch potential bugs and inefficiencies that `cargo check` would not.
+
+In summary:
+
+- Use `cargo check` when you want a fast way to check that your code compiles.
+- Use `clippy` when you want a more thorough analysis that includes style checks, code complexity, and potential logical errors.
+
+It's common to use both tools as part of your development workflow. You might run `cargo check` frequently while developing, and then run `clippy` periodically to clean up and refine your code.
```

Comment:
I feel like this whole section is really unnessecary.
I think you should instead add a link to the rust docs. 
What do you think? @mehalter

Reply:
I'll just make the decision.
I think this section is unnessecary, and should be removed.
Replace it with something like:

If you'd like to run cargo clippy instead of cargo check on safe, add the following...

@chaozwn

Reply:
> I'll just make the decision. I think this section is unnessecary, and should be removed. Replace it with something like:
> 
> If you'd like to run cargo clippy instead of cargo check on safe, add the following...
> 
> @chaozwn

fine